src/Components/Hero/Hero.tsx

```diff
@@ -0,0 +1,27 @@
+﻿import React from "react";
+import {Url} from "../../Models/Url";
+import './Hero.scss';
+
+
+interface HeroProps{
+    headingText : 'Curiosity' | 'Spirit'  | 'Opportunity';
+    rover : 'curiosity' | 'spirit'  | 'opportunity';
+    imageUrl : Url;
+}
+
+export function Hero(props : HeroProps){
+
+    const heroClass : string = "hero " + props.rover;
```

Comment:
Your version works just fine, but we can probably tidy this up a little by doing it all inline.

```jsx
<div className={`hero ${props.rover}`}>
```

Reply:
way better thanks